.gitignore

```diff
@@ -1,3 +1,7 @@
 .idea/
 target/
 *.iml
+/.settings/
```

Comment:
@amihaiemil there is no "projects/workspace" relationship in IDEA, you just open each project separately